docker-compose.yml

```diff
@@ -7,6 +7,10 @@ services:
       - '9090:8080'
     volumes:
     - ./target/scala-2.12/hmda.jar:/opt/hmda.jar
+    depends_on:
+      - query_db
+    environment:
+      JDBC_URL: jdbc:postgresql://192.168.99.100:54321/hmda?user=postgres&password=postgres
```

Comment:
I tried the configuration you suggest and it didn't work for me, so I ended up settling on this